hq/app/logic/DocumentUtil.scala

```diff
@@ -15,12 +15,11 @@ object DocumentUtil {
   private val parser = Parser.builder(options).build()
   private val renderer = HtmlRenderer.builder(options).build
 
-  def convert(markdownfile: String, sourceTransformation: String => String = identity): Option[Html] = {
```

Comment:
The only usage of `sourceTransformation` was a string substitution on the Snyk SSO link, so this argument is now redundant